usl_models/usl_models/atmo_ml/model_params.py

```diff
@@ -0,0 +1,32 @@
+"""AtmoML model parameters."""
+
+from dataclasses import dataclass
+
+import tensorflow as tf
+
+
+@dataclass(kw_only=True, slots=True)
+class AtmoModelParams:
+    # General parameters.
+    batch_size: int = 64
+
+    # Layer-specific parameters.
+    lstm_units: int = 512
+    lstm_kernel_size: int = 5
+    lstm_dropout: float = 0.2
+    lstm_recurrent_dropout: float = 0.2
+
+    # Keras parameters.
+    # Keras models support both Optimizer objects and (valid) string names.
+    # It is the user's responsibility to pass in a valid optimizer value.
+    optimizer: tf.keras.Optimizer | str = "adam"
+    epochs: int = 10
+
+
+# Used for testing.
+test_model_params = AtmoModelParams(
```

Comment:
Might suggest moving this definition into a file somewhere under the `tests/` directory to avoid mixing test & non-test definitions.

Reply:
I defined a helper function in the test file itself.  This will also make sure we use a different object every time, rather than using the same instance. (I don't think the previous behavior broke anything, but it probably wasn't the best setup. I'm planning on going and fixing the flood model in a separate PR.)